tests/test_utils.py

```diff
@@ -12,8 +12,8 @@
 from waffle_hub.utils.data import resize_image
 from waffle_hub.utils.evaluate import (
     evaluate_classification,
+    evaluate_instance_segmentation,
     evaluate_object_detection,
-    evaluate_segmentation,
 )
```

Comment:
We don't have test codes related to 'evalute_instance_segmentation' and 'evalute_semantic_segmentation', can you add them?